index.html

```diff
@@ -5,8 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" type="text/css" />
 </head>
+
 <body>
-    
+    <header>
+        <h2> in <span id="city" class="cityName">city</span>,</h2>
+        <h1>WEATHER REPORT</h1>
+    </header>
+
+    <main>
+        <section id="cityInput">
+            <h3> what is the current weather? : </h3>
+            <input id="cityText" class="cityTextBox"></input>
+            <button id="irlTemp" class="currentTempButton">irl temp</button>
+            <button id ="reset">☁☁reset☁☁</button>
+        </section>
+
+        <section id="weatherGarden"> 
+            <section id="sky" class="skyRainy"></section>
+            <section id="landScape" class="landTemperate"></section>
+        </section>
+        <div>
+            <section id="tempAdjust" class ="tempTemperate">
+                <h4>temp celcius</h4>
+                <div id ="tempBox">
+                    <button id="lowerTemp">👇</button>
+                    <h2 id="temp">23</h2>
```

Comment:
I'm not sure an `h2` would be appropriate nested under an `h4`. Headings should be used to help create the hierarchy of content on a page. Having an `h2` here may be confusing for readers using accessibility tools. I would recommend using something like a `p` element with CSS to set a larger font.